app/Resources/views/show/new.html.twig

```diff
@@ -5,19 +5,22 @@
 {% block body %}
 
 <h3>Create a new show</h3>
+
 <div class="panel">
-    <p>Required fields are marked with an asterisk (*). You can change any field later, with the
-    possible exception of Society: please contact us to change it later if neccessary.
-    Once the show is created you'll be able to add a
-    publicity image (please ensure you have permission to use it!).</p>
-
-    <p>Write a Description with the general public in mind – what is this show and why
-    should somebody come to see it? You can add auditions and production team vacancies
-    separately once the show is created; they don't need to be mentioned here.</p>
-    {% if not is_granted('APPROVE', form.vars.data) %}
-        <p>When you click 'create', it will be sent to a moderator, who it check it over before it
-        appears on Camdram.</p>
-    {% endif %}
+    <p> Please ensure you're familiar with our
+        <a href="{{ path('acts_camdram_userguidelines') }}">User
+        Guidelines</a> and
+        <a href="{{ path('acts_camdram_faq') }}#suitability">Show
+        Suitability Guidelines</a> first.
+    </p>
+
+    <p> Required fields are marked with an asterisk (*). You can change any field
```

Comment:
> You can change any field later, except possibly Societies

@GKFX do you know if this is still the case since `multisocs` was merged in? I seem to be able to edit a shows societies without obvious error on userguidelines.dev.camdram.net but could be missing something.

Reply:
Looking back through #538 anyone can currently edit societies so not much point referring to it specifically. Or maybe it helps to keep it ambiguous 🤔

Reply:
Yes I think we left it as open editing of societies while we worked out what, if any, restrictions were best. I don't think that discussion was really resolved so I'd avoid reference to it for now.

Reply:
Wording amended in 2e7927d to remove mention of societies as a special case.

Reply:
LGTM! 👍🚀💯